mathesar_ui/src/component-library/pagination/Pagination.svelte

```diff
@@ -112,39 +116,41 @@
               : `Goto Page ${_page}`}
             aria-selected={currentPage === _page}
             on:click={(e) => setPage(e, _page)}
+            on:mousedown={(e) => e.preventDefault()}
```

Comment:
What is the purpose of preventing default of mousedown event here?

Reply:
I think you're worried that the user might get confused on which one is the active one. It would be better to change the css of focus state instead, so that there is clear visual distinction on which one is focused and which one is active.

For the focus state, it would be better to add an outline instead of changing border color.

Reply:
@pavish I'll take them out. [This video](https://www.loom.com/share/84ce82efe06b4d1690f8cfbdaccf7cc4) explains the issue I was  seeing.

Reply:
@mr-gabe49 I think this should be fixed by separating out the css for active and focus states, as mentioned above.